test/libsolidity/syntaxTests/controlFlow/storageReturn/assembly/returning_function.sol

```diff
@@ -0,0 +1,13 @@
+contract C {
+    struct S { bool f; }
+    S s;
+    function f() internal pure returns (S storage c) {
+        // this should be an error, but currently function flow is ignored
```

Comment:
Is this a good example? I'm actually not sure if it should be an error, especially since the solidity function `f` is internal. The `return(0,0)` jumps beyond the point where the Solidity function `f` returns, so nobody will be able to use the uninitialized storage pointer.